services/keyword_service/keyword.go

```diff
@@ -62,31 +63,31 @@ func (k *KeywordService) Save(parsedKeywordList []string) error {
 	for _, value := range parsedKeywordList {
 		keyword := models.Keyword{Keyword: value, UserID: k.CurrentUserID}
 
-		txErr := db.GetDB().Transaction(func(tx *gorm.DB) error {
+		err := db.GetDB().Transaction(func(tx *gorm.DB) error {
 			savedKeyword, err := models.SaveKeyword(keyword, tx)
 			if err != nil {
 				return errorHandler.DatabaseErrorMessage(err)
 			}
 
-			enqueueErr := google_search_service.EnqueueSearchJob(savedKeyword)
-			if enqueueErr != nil {
-				return enqueueErr
+			err = google_search_service.EnqueueSearchJob(savedKeyword)
+			if err != nil {
+				return err
 			}
 
 			return nil
```

Comment:
@gutakk How about this?
```
return google_search_service.EnqueueSearchJob(savedKeyword)
```

Reply:
Fixed in f8132d480a676f9c4ccab60430f3efbe5f3b9f8c